content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,177 @@
+  Rekey Configuration and Usage
+---
+
+## Rekey Controller Installation
+
+The CSM Encryption Rekey Controller must be installed in order to be able to perform key rotation on CSM encrypted volumes in a
+Kubernetes cluster. The Rekey Controller can be installed via the standard Dell Helm Chart repository available at  https://github.com/dell/helm-charts.git.
+
+A secret of the cluster config must be created with the name ``cluster-kube-config`` typically from the .kube/config. Here is an example:  
+
+```
+kubectl create secret generic cluster-kube-config --from-file=/home/root/.kube/config
+```
+
+The `csm-encryption-rekey-controller/values.yaml` must be adapted to the kubernetes cluster via the `provisioner` and `port` parameters.
+Here is an example of a typical values.yaml file:
+
+```yaml
+# Rekey controller image name.
+image: dellemc/csm-encryption-rekey-controller:v0.1.0
+
+# Rekey controller image pull policy.
+# Allowed values:
+#  Always: Always pull the image.
+#  IfNotPresent: Only pull the image if it does not already exist on the node.
+#  Never: Never pull the image.
+imagePullPolicy: IfNotPresent
+
+# This value is required and must match encryption.pluginName value
+# of the corresponding Dell CSI driver.
+provisioner: sec-isilon.dellemc.com
+# This represents the port number used by encryption driver Rest API server.
+port: 3838
+```
+> **NOTE**: It is important that the `provisioner` reflect the same name as the `pluginName` specified in the standard encryption values.yaml file, and the `port`
+reflect the same port number as the encryption driver REST API server would be listening on. If they don't match then all rekey attempts will fail.
+
+## Deploy Rekey Controller
+
+Copy the values.yaml to a local file. Once the CSM encryption Rekey Controller local values.yaml file has been
+adjusted for the current cluster, deploy the controller by installing
+the helm chart. As an example:
+
+``` helm install --values local-values.yaml isilon-rekey csm-encryption-rekey-controller```
+
+A rekey-controller pod should now be up and running.
+
+
+## Rekey Controller Usage
+
+The CSM Encryption Rekey Controller must be installed on a Kubernetes cluster in order to be able to Rekey encrypted 
+volumes. The general procedure for rekeying is to create a rekey custom resource via a simple yaml configuration. This 
+will kick off a rekey process on the PV specified as the `volume` in the resource. 
+
+## Rekey with dellctl
+
+If dellctl CLI is installed, rekey'ing a volume is simple. First, identify a volume with a PV that is encrypted with the CSM encryption driver
+provisioner. Verify that there is at least one volume attachment for this PV.
+
+For example, let's rekey an encrypted PV with the name ``K8s-asdf1234``:
+
+``dellctl encryption rekey myrekey k8s-asdf1234``
+
+```
+INFO rekey request "myrekey2" submitted successfully for persistent volume "k8s-112a5d41bc".
+INFO Run 'dellctl encryption rekey-status myrekey2' for more details.

```

Comment:
The rekey command was for "myrekey", not "myrekey2". PV name is different too.

Reply:
fixed the names.